packages/netlify-cms-core/src/types/redux.ts

```diff
@@ -401,7 +400,7 @@ export interface CmsConfig {
   media_folder_relative?: boolean;
   media_library?: CmsMediaLibrary;
   publish_mode?: CmsPublishMode;
-  default_workflow_status?: PublishStatusValues;
+  default_workflow_status?: string;
```

Comment:
With this change, the `default_workflow_status` will just need to be a string, and it can be value, correct?

If you want it to be one of `pending_review`, `pending_publish` or `draft` (values of the object `Statues`), can you do this?

```
type ValueOf<T> = T[key of T]
inteface Config {
  default_workflow_status?: StatusValues
}

```

Reply:
The line you suggest will give a string value no ? It's better to have an enum ?

Reply:
It's okay for you @bytrangle : 
- https://github.com/netlify/netlify-cms/pull/6469/commits/08255b6168769a562e827049af0ebd4535862c34

![image](https://user-images.githubusercontent.com/16322590/172157127-b641dd1c-5bd0-4e23-99ea-9ae835a4d91f.png)

Reply:
I missed a line in my suggested code.

```
type ValueOf<T> = T[keyof T];

type StatusValues = ValueOf<Statues>

interface Config {
    default_workflow_status?: StatusValues
}
```

I'm not fluent in Typescript, but the setting above will force `default_workflow_status` to be any of `draft`, `pending_review` and `pending_publish`, no?

But like you said, it won't prevent users from inputting an invalid value.

I'm fine with what you've done, but since this is a feature, I'm awaiting another reviewer's feedback.

Reply:
Yes your suggestion it's the same as the new type I created.

To prevent the user from inputting an invalid value there is the method in applyDefault :) 

Thanks to you for your help ;)

I will wait for another review